### PR TITLE
feat: support for cloudwatch config on multi-runners

### PR DESCRIPTION
**Pull request by piscue**
## Problem
Cloud config can only set for all multi-runners and not per runner. 

## Solution
Add option to set the cloud config per runner.

### CODE REVIEW

**Comment by piscue** [comment]
@npalm could you take a look on this one?

File: modules/multi-runner/runners.tf [hunk 1]
```diff
@@ -69,7 +69,7 @@ module "runners" {
   logging_retention_in_days        = var.logging_retention_in_days
   logging_kms_key_id               = var.logging_kms_key_id
   enable_cloudwatch_agent          = each.value.runner_config.enable_cloudwatch_agent
-  cloudwatch_config                = var.cloudwatch_config
+  cloudwatch_config                = each.value.runner_config.cloudwatch_config
```
Comment (npalm) [inline modules/multi-runner/runners.tf:72]: Can you change this to something like

```coalesce(each.value.runner_config.cloudwatch_config, var.cloudwatch_config)```


And update the docs of both variables/

**Review by npalm — CHANGES_REQUESTED** [review]
Looks good, see comment with a suggestion

File: modules/multi-runner/runners.tf (hunk 1 above)
Comment (piscue) [reply modules/multi-runner/runners.tf]: I did the update and the doc I've found. The rest of the documentation seems to exist already.

**Review by npalm — COMMENTED** [review]
Looks god, wil run asap a quick test

**Review by npalm — APPROVED** [review]

**Pull request merged**